tests/drivers/test_elasticsearch.py

```diff
@@ -344,6 +344,58 @@ def test_fetch_points_updates_read_on(self):
         # Try again, we read_on != None now
         self.accessor.fetch_points(metric, 0, 1, metric.retention[0])
 
+    def test_metric_out_of_search_bounds_should_not_be_found(self):
+        metric_name = "test_metric_out_of_search_bounds_should_not_be_found.a.b.c"
+        metric = self.make_metric(metric_name)
+        self.accessor.create_metric(metric)
+        self.accessor.flush()
+
+        now = datetime.datetime.now()
+        one_week = datetime.timedelta(weeks=1)
+        bounds = [
+            (now - 2 * one_week, now - one_week),
+            (now + one_week, now + 2 * one_week),
+            (None, now - one_week),
+            (now + one_week, None)
+        ]
+
+        for start_time, end_time in bounds:
+            # Without bounds, metric should be found (1 element in the iterator)
+            should_be_found_metrics = self.accessor.glob_metric_names(metric_name)
+            self.assert_iter_not_empty(should_be_found_metrics)
```

Comment:
I understand why you do it but shouldn't it already be covered by other tests ? If you can remove this, you could factorize both unit tests

Reply:
Done.